cmd/prepare-storage.go

```diff
@@ -312,9 +319,14 @@ func waitForFormatErasure(firstDisk bool, endpoints Endpoints, poolCount, setCou
 	}
 
 	// Wait on each try for an update.
-	ticker := time.NewTicker(500 * time.Millisecond)
+	ticker := time.NewTicker(250 * time.Millisecond)
```

Comment:
I guess you can set this to one millisecond and then run ticker.Reset(250*time.Millisecond) at the end of the next for block.

Reply:
also its not a timer and doesn't need a reset @vadmeste